### ISSUE TITLE
Headings following tables cause badly-nested HTML

### ISSUE THREAD

**Issue by quis** [alphagov/tech-docs-template#148]
Test case in the before diff here: https://github.com/alphagov/notifications-tech-docs/pull/22

---

Looks like this (note the table appearing below the footer):

![image](https://user-images.githubusercontent.com/355079/38920285-d022f3c0-42ea-11e8-9f0a-e3b6f84dada8.png)

---

Due to invalid HTML, as detected by the validator:

![image](https://user-images.githubusercontent.com/355079/38920455-361a39cc-42eb-11e8-9644-bb4f5cff2daf.png)

---

Because there is no closing `</table>` tag following the `</tr>`:

![image](https://user-images.githubusercontent.com/355079/38920592-8494d5b2-42eb-11e8-80bf-077940101f5c.png)

**Comment by bahmady** [comment]
I don't think this only affects headings after tables. I've hit upon this a few times and it seems to come up "randomly". Often tables which are unaffected will just _become_ affected after an update to the tech docs tool. These are recent screenshots from the gds-way docs:

![table_bug2](https://user-images.githubusercontent.com/32935794/46481801-14cd2c80-c7ec-11e8-8620-a2dc3783c9f3.png)
![table_bug1](https://user-images.githubusercontent.com/32935794/46481805-172f8680-c7ec-11e8-90c7-a546ec3ddb23.png)

As a quick and dirty workaround I've been putting this line *before and after* all tables (to be on the safe side) in Markdown:

`<div style="height:1px;font-size:1px;">&nbsp;</div>`

**Comment by soupdragon99** [comment]
The attached text file contains the original markdown for the gds-way doc @bahmady mentions

[supporting-services.txt](https://github.com/alphagov/tech-docs-template/files/2449791/supporting-services.txt)
